Tests/TestStateManager.swift

```diff
@@ -244,4 +244,22 @@ class TestStateManager: XCTestCase {
             )
         )
     }
+    
+    @available(iOS 13, macOS 13, watchOS 13, tvOS 13, *)
```

Comment:
Too many 13s here :)
For Task the requirements are written here https://developer.apple.com/documentation/swift/task
tvOS and iOS can be ok, but macOS and watchOS need to be lowered.